result.go

```diff
@@ -381,17 +381,42 @@ func (q Query) ToMapBy(
 	res.Elem().Set(m)
 }
 
-// ToSlice iterates over a collection and populates the result slice with elements.
+// ToSlice iterates over a collection, copy to result slice the collection elements,
```

Comment:
Looks great for me @ahmetalpbalkan 👍 , thanks for helping me with documentation.